src/functions/Output.ps1

```diff
@@ -43,6 +43,7 @@ $script:ReportTheme = DATA {
         PassTime         = 'DarkGray'
         Fail             = 'Red'
         FailTime         = 'DarkGray'
+        FailMessage      = 'Gray'
```

Comment:
If the suggestions of what to color darker are implemented this would be stack trace. Maybe just call it FailDetail?

Reply:
changed